plugins/required-file/index.js

```diff
@@ -95,8 +95,7 @@ class RequiredFilePlugin extends BasePlugin {
      * @type {PullsListFilesResponseItem[]}
      */
     const filesInPR = await context.github.paginate(context.github.pulls.listFiles.endpoint.merge({
-      ...context.repo(),
-      pull_number: context.payload.number
+      ...context.pullRequest()
     }), res => res.data);
```

Comment:
since you're not adding in additional data now, should this just be:

```js
    const filesInPR = await context.github.paginate(
      context.github.pulls.listFiles.endpoint.merge(context.pullRequest()),
      res => res.data);
```